src/Business Foundation/App/AuditCodes/app.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id":  "88dda422-d220-415c-bd1c-6240e5a9ceee",
+    "name":  "Audit Codes",
+    "publisher":  "Microsoft",
+    "brief":  "",
+    "description":  "",
+    "version":  "25.0.0.0",
+    "privacyStatement":  "https://go.microsoft.com/fwlink/?linkid=724009",
+    "EULA":  "https://go.microsoft.com/fwlink/?linkid=2009120",
+    "help":  "https://go.microsoft.com/fwlink/?linkid=2103698",
```

Comment:
We need to get this updated. The help currently links to the `system application`'s main page. The `business foundation`'s help need to be updated as well.

Reply:
yup, nice catch

Reply:
taling to Brent and Susanne ... I think we'll reword the docs pages so they apply more genericly... in any case, updating the docs will take take us a while so opened a doc bug for it